python_examples/sinric/_cbhandler.py

```diff
@@ -0,0 +1,32 @@
+
+class CallBackHandler:
+    def __init__(self, callbacks):
+        self.callbacks = callbacks
+
+    def handle_callbacks(self, jsn):
+        if jsn['action'] == 'setPowerState':
+            try:
+                power_callback = self.callbacks['setPowerState']
+                power_callback(jsn['deviceId'], jsn['value'])
+            except Exception as e:
+                print(e)
```

Comment:
Perhaps instead of `print` statements, use [loguru](https://github.com/Delgan/loguru)